File: java/logica_y_validaciones/validar_nombre_and_telefono.java
package logica_y_validaciones;
public class validar_nombre_and_telefono {
    public static void main(String[] args) {
        String nombre = "Ana Gómez";
        String telefono = "3001234567";
        if (!nombre.isEmpty() && telefono.length() == 10) {
            System.out.println("Nombre y teléfono son válidos.");
        } else {
            System.out.println("Error: nombre o teléfono inválido.");
        }
    }
}

File: visual_basic/operaciones_logicos/validar_nombre_and_telefono/validar_nombre_and_telefono.vb
Imports System
Module validar_nombre_and_telefono
    Sub Main(args As String())
        Dim nombre As String
        Dim telefono As String
        Console.Write("Ingrese el nombre: ")
        nombre = Console.ReadLine()
        Console.Write("Ingrese el teléfono: ")
        telefono = Console.ReadLine()
        If nombre.Trim() <> "" And telefono.Trim() <> "" Then
            Console.WriteLine("Datos válidos ✅")
            Console.WriteLine("Nombre: " & nombre)
            Console.WriteLine("Teléfono: " & telefono)
        Else
            Console.WriteLine("Error: Debe completar nombre y teléfono ❌")
        End If
    End Sub
End Module